main.go

```diff
@@ -67,21 +68,32 @@ func main() {
 func doCheckExistenceAndParse() {
 	switch {
 	case strings.Contains(path, "Gopkg.lock"):
-		dep := packages.Dep{}
-		dep.GopkgPath = path
-		if dep.CheckExistenceOfManifest() {
-			dep.ProjectList, _ = parse.GopkgLock(path)
-			var purls = processPackages(dep)
-			var packageCount = len(purls)
+		slices := strings.Split(path, "/Gopkg.lock")
```

Comment:
I think you might be able to do `filepath.Dir(path)`

https://golang.org/pkg/path/filepath/#Dir